<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>531</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QFrame" name="topbar_frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="title_label">
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Build An Asset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="help_pushButton">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="tooltips_label">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>Tool Tips</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="main_frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QTabWidget" name="main_tabWidget">
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QTabBar::tab { 
	height: 100px; width: 100px;
	 }
QTabBar {
	padding: 3 3px;
	}

QTabBar::tab:first { 
	image: url(:/icons/mat-common/mat/common/resources/icons/tools/jinx_icon.png);
	 }


QTabBar::tab:middle { 
	image: url(:/icons/mat-common/mat/common/resources/icons/tools/herbert_icon.png)
	 }


QTabBar::tab:last { 
	image: url(:/icons/mat-common/mat/common/resources/icons/tools/popcat_icon.png);
	 }

</string>
            </property>
            <property name="tabPosition">
             <enum>QTabWidget::West</enum>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::Rounded</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="elideMode">
             <enum>Qt::ElideNone</enum>
            </property>
            <property name="tabsClosable">
             <bool>false</bool>
            </property>
            <property name="movable">
             <bool>false</bool>
            </property>
            <widget class="QWidget" name="model_buttontab">
             <attribute name="title">
              <string>Model</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QTabWidget" name="model_tabWidget">
                <property name="styleSheet">
                 <string notr="true">QTabBar::tab { 
	height: 20px; width: 50px;
	padding: 0 8px;
	
	image: url(:/icons/mat-common/mat/common/resources/icons/tools/jinx_icon.png);
 }</string>
                </property>
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="bruh_tab">
                 <attribute name="title">
                  <string>bruh</string>
                 </attribute>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <layout class="QHBoxLayout" name="helios_Layout">
                    <property name="sizeConstraint">
                     <enum>QLayout::SetDefaultConstraint</enum>
                    </property>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="helios2_Layout">
                    <item>
                     <widget class="QLabel" name="heliosimage_Label">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="materials_tab">
                 <attribute name="title">
                  <string>Materials</string>
                 </attribute>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QPushButton" name="applymats_pushButton">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Apply Materials to Selected</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="skeleton_buttontab">
             <attribute name="title">
              <string>Skeleton</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="rig_buttontab">
             <attribute name="title">
              <string>Rig</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QTabWidget" name="tabWidget">
                <property name="styleSheet">
                 <string notr="true">QTabBar::tab { 
	height: 20px; width: 70px;
	padding: 0 8px;
	
	image: url(:/icons/mat-common/mat/common/resources/icons/tools/jinx_icon.png);
 }</string>
                </property>
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <widget class="QWidget" name="bar_tab">
                 <attribute name="title">
                  <string>BAR</string>
                 </attribute>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <layout class="QHBoxLayout" name="bar_Layout"/>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="rigsettings_tab">
                 <attribute name="title">
                  <string>Rig Settings</string>
                 </attribute>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <layout class="QVBoxLayout" name="rigsettings_Layout">
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>Temp box for rig settings</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="flagrotationorder_Layout"/>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="animation_buttontab">
             <attribute name="title">
              <string>Animation</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../ban_icons.qrc"/>
 </resources>
 <connections/>
</ui>
